pythonforandroid/recipes/pygame_bootstrap_components/__init__.py

```diff
@@ -25,5 +25,11 @@ def prebuild_arch(self, arch):
         info('Unpacking was successful, deleting original container dir')
         shprint(sh.rm, '-rf', self.get_build_dir(arch))
 
+        info('Applying jpeg assembler patch')
+        ndk_15_plus_patch = join(self.get_recipe_dir(), 'jpeg-ndk15-plus.patch')
+        shprint(sh.patch, '-t', '-d',
+                join(self.get_build_container_dir(arch), 'jpeg'), '-p1',
+                '-i', ndk_15_plus_patch, _tail=10)
```

Comment:
Just wondering why don't we apply the patch using the `patches` attribute?

Reply:
As far as I remember...it's for the jpeg path, If we use the default patching method the file will not be found...but I'm thinking now that maybe we can add the needed levels to the patch, so we could refactor this into the default patching mechanism...I suppose that this should work...I will try to make some tests about that ;)

Reply:
Yes that would be awesome, that would simplify the code a little bit :+1: